zetaclient/orchestrator/orchestrator.go

```diff
@@ -618,7 +618,7 @@ func (oc *Orchestrator) ScheduleCctxSolana(
 		oc.logger.Error().Msgf("ScheduleCctxSolana: chain observer is not a solana observer")
 		return
 	}
-	// #nosec G701 positive
+	// #nosec G115 G701 positive
```

Comment:
typo?

Reply:
no, but it is incorrect. G701 was removed. Updated.